pallets/auctions/src/lib.rs

```diff
@@ -194,13 +220,26 @@ pub mod pallet {
 	#[pallet::hooks]
 	impl<T: Config> Hooks<T::BlockNumber> for Pallet<T> {
 		fn on_finalize(now: T::BlockNumber) {
-			Self::conclude_auction(now);
+			Self::conclude_auction(now).unwrap_or_default();
 		}
 	}
 }
 
 impl<T: Config> Pallet<T> {
-	fn conclude_auction(now: T::BlockNumber) {
+	fn validate_update_delete(
```

Comment:
The name is confusing to me . ? What it does - validates, updates or deletes?:-)

According to implementation - i guess it is something like "can_delete" ?  If so, it does not need DispatchResult, simple bool type would suffice.